CMakeLists.txt

```diff
@@ -14,8 +14,14 @@ endif()
 
 # global settings
 
+#everything's correct, moving on.
+set(CMAKE_MODULE_PATH
+        ${CMAKE_MODULE_PATH}
+	${${PROJECT_NAME}_SOURCE_DIR}/cmake/Modules
```

Comment:
this looks like there might be a whitespace issue

Reply:
also, this adds our path after the existing module path. so that means that if we have a name conflict, we might now pick up a different cmake module... the explicit paths that you are changing in the rest of this patch would have prevented that... so would it make more sense to prepend our path?

Reply:
Fixed both.